packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -92,13 +92,15 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           onClick={ this._onDropdownClick }
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
+          aria-live={ disabled || isOpen ? 'off' : 'assertive' }
```

Comment:
Likely because everywhere people are using assertive which interrupts the output stream.